.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: CI workflow
+on: [pull_request]
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
```

Comment:
why run in a matrix when you just have 1 OS?

Reply:
copy paste